divvy/cloudfilebrowser.js

```diff
@@ -629,6 +641,11 @@ var server = (function() {
                 elementProvision.configuration["document.tagging"] = true;
                 elementProvision.configuration["filter.response.nulls"] = true;
                 elementProvision.configuration["onedrivebusiness.site.address"] = cbArgs.elementDetails.siteAddress;
+            } else if (element === 'sharepoint') {
+                elementProvision.configuration["oauth.scope"] = "AllSites.Manage";
```

Comment:
```suggestion
                elementProvision.configuration["oauth.scope"] = "Site.Read";
```
Also, because of this we should remove the search box for Sharepoint, and the ability to drop files in the overlay.